slurm_tmpdir.py

```diff
@@ -0,0 +1,78 @@
+"""This file creates a new data file like train.json in slurm_tmpdir
```

Comment:
Does this just create a copy of the file ?

Reply:
creates a copy of the file with updated paths to use slurm_tmpdir but as discussed with @51N84D it does not change much so probably a waste of my time :p